### ISSUE TITLE
"Zoom to Layer" Functionality 

### ISSUE THREAD

**Issue by not-Karot** [keplergl/kepler.gl#2446]
**Is your feature request related to a problem? Please describe.**

When using kepler.gl with multiple datasets, it's currently challenging to focus on a specific dataset without manually zooming out entirely and then navigating back in. This process is inefficient and time-consuming, especially compared to standard GIS tools that offer a "Zoom to Layer" functionality.

**Describe the solution you'd like**

Introduce a "Zoom to Layer" option in kepler.gl, allowing users to quickly zoom in on a selected dataset or layer, enhancing usability and aligning kepler.gl with common GIS platform features.

**Additional context**
The absence of a direct "Zoom to Layer" feature is quite noticeable, as this is a fundamental operation in most GIS platforms, enabling users to focus on specific layers or datasets instantly.

**Comment by liuwo23** [comment]
I can implement this functionality, but I'm not sure if it will be adopted and integrated into the source code.

**Comment by ibgreen** [comment]
> I can implement this functionality, but I'm not sure if it will be adopted and integrated into the source code.

Yes... It is best to describe your approach first before opening any non-trivial PR, so the maintainers can indicate if they think they can get behind the changes.  Discussions tab could be a good place.

**Comment by liuwo23** [comment]
> u

Okay, I will push a PR.

**Comment by liuwo23** [comment]
> > I can implement this functionality, but I'm not sure if it will be adopted and integrated into the source code.
> 
> Yes... It is best to describe your approach first before opening any non-trivial PR, so the maintainers can indicate if they think they can get behind the changes. Discussions tab could be a good place.

Okay, I will push a pull request.
